### PR TITLE
Add E2E Tests

### PR DESCRIPTION
**Pull request by JanusWu**
Issue: #142, #143
Add End To End tests to this repo. And I will add the tests in batches.

### CODE REVIEW

**Comment by JanusWu** [comment]
Only after we make sure there are no problems in previous tests, then I will commit another batch of tests.  I think this will make this PR more clear.

File: End2End/HTTP/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.microsoft.azure.functions</groupId>
+                <artifactId>azure-functions-java-library</artifactId>
+                <version>1.0.0-beta-4</version>
```
Comment (pragnagopa) [inline End2End/HTTP/pom.xml]: Move to latest version of azure-functions-java-library 1.0.0-beta-5

File: End2End/HTTP/pom.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (pragnagopa) [inline End2End/HTTP/pom.xml]: Having pom.xml for each test does not look right.  You need to combine tests
- One pom.xml
- All the java functions should be in one Jar

**Comment by pragnagopa** [comment]
Rebase on dev branch and when you send an updated PR

File: End2End/HTTP/pom.xml (hunk 2 above)
Comment (JanusWu) [reply End2End/HTTP/pom.xml]: Sure, I will combine the tests.

**Comment by pragnagopa** [comment]
Looks good. I will merge the changes. Please send a PR for the rest of the tests.

**Pull request merged**